### PR TITLE
Playwright test for Attributes Filtering

### PR DESCRIPTION
**Pull request by sievdokymov-virtru**
https://virtru.atlassian.net/browse/PLAT-1777

### CODE REVIEW

File: tests/e2e/Attributes.spec.ts [hunk 1]
```diff
@@ -11,8 +11,7 @@ test.describe('<Attributes/>', () => {
     await page.locator(selectors.tokenMessage).click()
     await createAuthority(page, authority);
     // click success message to close it and overcome potential overlapping problem
-    // fixme: error page.locator(selectors.alertMessage) matches 2 elements instead of 1
```
Comment (sievdokymov-virtru) [inline tests/e2e/Attributes.spec.ts:14]: Test passes successfully.

2 messages appear when Authority Creation is failed due to Name duplication case (possible case for multiple test attempts). So overall test failure is indicated properly in this case.

I've additionally increased range for Name randomiser to decrease this probability.

File: tests/e2e/Attributes.spec.ts [hunk 2]
```diff
@@ -25,12 +24,43 @@ test.describe('<Attributes/>', () => {
     test.expect(newAuthority).toBeTruthy();
   });
 
-  test('should add attribute', async ({ page, attributeName, authority, attributeValue }) => {
+  test('should add attribute' + ',should filter attributes by Name, Order', async ({ page, attributeName, authority, attributeValue }) => {
```
Comment (ivanovSPvirtru) [inline tests/e2e/Attributes.spec.ts:27]: nitpicking, its probably some IDE accidental stuff)

**Review by ivanovSPvirtru — APPROVED** [review]
lgtm

File: tests/e2e/Attributes.spec.ts (hunk 2 above)
Comment (sievdokymov-virtru) [reply tests/e2e/Attributes.spec.ts:27]: nope, formed intentionally that way to look better in the report)

File: tests/e2e/helpers/fixtures.ts [hunk 3]
```diff
@@ -10,7 +10,7 @@ export const test = baseTest.extend<{ attributeName: string; authority: string;
     await use(attributeName);
   },
   authority: async ({ page }, use) => {
-    const authority = `https://opentdf${generateRandomDigit(1000, 1)}.ua`;
+    const authority = `https://opentdf${generateRandomDigit(10000, 1)}.ua`;
```
Comment (obidenko) [inline tests/e2e/helpers/fixtures.ts:13]: Go for 10 000 000 , should be enough

File: tests/e2e/Attributes.spec.ts [hunk 4]
```diff
@@ -25,12 +24,43 @@ test.describe('<Attributes/>', () => {
...
+    await page.fill(selectors.attributesPage.attributesHeader.filterModal.nameInputField, 'invalidAttributeName')
+    await page.click(selectors.attributesPage.attributesHeader.filterModal.submitBtn)
+    await expect(page.locator(selectors.attributesPage.attributesHeader.itemsQuantityIndicator)).toHaveText('Total 0 items')
+
+    // filter by Order
+    await page.click(selectors.attributesPage.attributesHeader.filterModal.clearBtn)
+    await page.fill(selectors.attributesPage.attributesHeader.filterModal.orderInputField, attributeValue)
+    await page.click(selectors.attributesPage.attributesHeader.filterModal.submitBtn)
+    await page.click(selectors.attributesPage.attributesHeader.itemsQuantityIndicator)
+    const filteredAttributesListByOrder = await page.$$(selectors.attributesPage.attributeItem)
```
Comment (obidenko) [inline tests/e2e/Attributes.spec.ts:54]: small update : 

    const attributesHeader = selectors.attributesPage.attributesHeader;
    const filterModal = attributesHeader.filterModal;

    // filter by existed Name
    await page.click(attributesHeader.filtersToolbarButton)
    await page.fill(filterModal.nameInputField, attributeName)
    await page.click(filterModal.submitBtn)
    await page.click(attributesHeader.itemsQuantityIndicator)
    const filteredAttributesListByName = await page.$$(selectors.attributesPage.attributeItem)
    expect(filteredAttributesListByName.length).toBe(1)

    // filter by non-existed Name
    await page.click(attributesHeader.filtersToolbarButton)
    await page.click(filterModal.clearBtn)
    await page.fill(filterModal.nameInputField, 'invalidAttributeName')
    await page.click(filterModal.submitBtn)
    await expect(page.locator(attributesHeader.itemsQuantityIndicator)).toHaveText('Total 0 items')

    // filter by Order
    await page.click(filterModal.clearBtn)
    await page.fill(filterModal.orderInputField, attributeValue)
    await page.click(filterModal.submitBtn)
    await page.click(attributesHeader.itemsQuantityIndicator)
    const filteredAttributesListByOrder = await page.$$(selectors.attributesPage.attributeItem)

**Review by obidenko — CHANGES_REQUESTED** [review]

File: tests/e2e/Attributes.spec.ts (hunk 4 above)
Comment (sievdokymov-virtru) [reply tests/e2e/Attributes.spec.ts:57]: @obidenko 
Good idea! Thx.
Updated.

File: tests/e2e/helpers/fixtures.ts (hunk 3 above)
Comment (sievdokymov-virtru) [reply tests/e2e/helpers/fixtures.ts]: Updated.

**Review by obidenko — APPROVED** [review]

**Pull request merged**